src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -148,9 +151,27 @@ public bool TryGetTelemetryId(out Guid telemetryId)
                 }
             }
 
-            public IEnumerable<SuggestedActionSet> GetSuggestedActions(
+            public IEnumerable<SuggestedActionSet>? GetSuggestedActions(
+                ISuggestedActionCategorySet requestedActionCategories,
+                SnapshotSpan range,
+                CancellationToken cancellationToken)
+                => GetSuggestedActions(requestedActionCategories, range, addOperationScope: null, cancellationToken);
+
+            public IEnumerable<SuggestedActionSet>? GetSuggestedActions(
+                ISuggestedActionCategorySet requestedActionCategories,
+                SnapshotSpan range,
+                IUIThreadOperationContext operationContext)
+            {
+                return GetSuggestedActions(requestedActionCategories, range, AddOperationScope, operationContext.UserCancellationToken);
+
+                IDisposable AddOperationScope(string description)
+                    => operationContext.AddScope(allowCancellation: true, string.Format(EditorFeaturesWpfResources.Gathering_Suggestions_0, description));
```

Comment:
why the local function just to pass along?  why not just pass the lambda along as an arg to GetSuggstedActions?